scripts/daily_snapshot/upload_snapshot.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# If Forest hasn't synced to the network after 30 minutes, something has gone wrong.
+SYNC_TIMEOUT=30m
+
+CHAIN_NAME=$1
+NEWEST_SNAPSHOT=$2
```

shall we have a guard clause, failing early if arguments are not provided?